specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManagerSecurityUserConfiguration.json

```diff
@@ -426,6 +426,60 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/securityUserConfigurations/{configurationName}/rules": {
```

Comment:
You should expose both GET collection and GET on a resource type.
Add below API too
GET /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/securityUserConfigurations/{configurationName}/rules/{ruleName}

Refer section 'Inconsistent or incomplete CRUD' in https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html